tutorials/vscImportingGradleProject.md

```diff
@@ -9,21 +9,77 @@
-* **Open the project in VS Code**, as follows:
-  * Open VS Code.{{ abcd_numbers }}
-  * If you are in the welcome screen, Click `Open`. Otherwise, click `File` -> `Open`.
-  * Select the project directory, and click `Open`.
-* **Confirm the correct JDK is set** to the one you are supposed to use for the project, as follows:
-  * Open the Command Palette: {{ icon_windows }}/{{ icon_linux}} `Ctrl+Shift+P` | {{ icon_apple }} `Cmd+Shift+P` {{ abcd_numbers }}
-  * Type `Java: Configure Java Runtime` and select it.
-  * Check that the JDK version matches the one required for your project. If not, you can install the required JDK and configure it. (Java installation guides: [Windows](https://se-education.org/guides/tutorials/javaInstallationMac.html) | [Mac](https://se-education.org/guides/tutorials/javaInstallationWindows.html))
-* **Confirm the correct JVM is used for Gradle**, as follows:
-  * Install the Gradle Extension: Search for `Gradle` in the Extensions view (`Ctrl+Shift+X` or `Cmd+Shift+X`), and install a Gradle extension if you haven’t already (e.g., [Gradle for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-gradle)). {{ abcd_numbers }}
-  * Open `settings.json`: You can find it by opening the Command Palette (`Ctrl+Shift+P` or `Cmd+Shift+P`), then typing `Preferences: Open User Settings (JSON)`.
-  * Add or modify the following settings to configure the JVM for Gradle. Here's an example (for Mac):
+<box type="info" seamless>
+
+This tutorial is adapted from the [official VS Code Java tutorial for Gradle build tool](https://code.visualstudio.com/docs/java/java-build#_gradle).
+</box>
+
+This guide will help you import and work with existing Gradle projects in VS Code.
+
+**If you are new to working with Gradle in VS Code**, we recommend watching the video below for an introduction to the features of VS Code Gradle extension. 
+
+<panel header=":fab-youtube: Get started with VS Code Gradle extension" peek >
+
+@[youtube](dfiaqZX-504)
+
+</panel>
+
+**To recall how to use a specific feature**, you can refer to the sections below.
+
+## Prerequisites
+
+<box type="tip" seamless>
+
+**Need help with following prerequisites?** Check out our [Preparing VS Code for Java](vscPreparingForJava.html) guide first.
+</box>
+
+Before proceeding, ensure the following are installed:
+
+* VS Code with the [Extension Pack for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-java-pack), which includes the [Gradle for Java](https://marketplace.visualstudio.com/items?itemName=vscjava.vscode-gradle) extension
+* Java Development Kit (JDK)
+* Verify the project has support for Gradle. If it does, there should be a `build.gradle` file in your project root.
+
+## Importing an existing Gradle project
+
+### Step 1: Open the project folder
+
+1. **Open VS Code**
+2. **Select** `File` → `Open Folder...`
+3. **Navigate to your existing Gradle project** folder
+4. **Click "Select Folder"** to open it as a workspace
```

Comment:
![image](https://github.com/user-attachments/assets/37a42ce6-bf5c-411d-9ad9-7237b350892b)

See https://se-education.org/guides/conventions/markdown.html for more